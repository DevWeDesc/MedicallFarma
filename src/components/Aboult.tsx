import React from "react";
import ImagesGrid from "../../public/ImagesGrid.png";
import Image from "next/image";

export const Aboult = () => {
  return (
    <div className="flex flex-col-reverse py-12 px-5 gap-5 lg:pt-0 lg:px-0 lg:grid lg:grid-cols-2 lg:gap-20 lg:py-0">
      <Image alt="" className="lg:min-h-[30rem] lg:w-full" src={ImagesGrid} />
      <div className="flex flex-col gap-3 lg:py-8 lg:pr-16 lg:gap-10">
        <div className="flex flex-col lg:gap-5">
          <h2 className="text-xl font-bold lg:text-5xl">
            Sobre a Medicall Farma
          </h2>
          <div className="text-xs font-light lg:text-lg">
            <p>
              Atuamos há mais de 10 anos no mercado de vendas e distribuição de
              materiais médicos e medicamentos hospitalares em todo o estado de
              São Paulo.
            </p>
            <p>
              Oferecemos a melhor experiência de atendimento aos nossos
              clientes, fornecedores e acionistas, dede o primeiro contato.
              Trabalhamos com um mix de Produtos.
            </p>
            <p>
              Diversificados das melhores marcas nacionais e internacionais
              garantimos total adilidade e pontualidade nas entregas através de
              um rigoroso processo de qualidade operacional.
            </p>
          </div>
        </div>
        <div className="flex flex-col lg:gap-5">
          <h2 className="text-xl font-bold lg:text-5xl">Nossa Missão</h2>
          <p className="text-xs font-light lg:text-lg">
            A Medicall Farma tem como missão contribuir com a melhoria da vida e
            saúde das pessoas, por meio de fornecimento de materiais médicos e
            medicamentos hospitalares de alta qualidade às instituições de saúde
            privadas e públicas do estado de São Paulo.
          </p>
        </div>
      </div>
    </div>
  );
};
