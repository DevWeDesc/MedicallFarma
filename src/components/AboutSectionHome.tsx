import React from "react";
import imageMedicall from "../../public/imagemedicall.jpeg";
import Image from "next/image";
import Link from "next/link";
import { AnimationScroll } from "@/animations/AnimationScroll";

export const AboutSectionHome = () => {
  return (
    <AnimationScroll>
      <div className="flex flex-col justify-center relative items-center bg-black text-white">
        <Image
          alt=""
          className="w-full object-cover h-80 opacity-60"
          src={imageMedicall}
        />
        <div className="absolute h-full w-full flex flex-col justify-center gap-8 items-center">
          <p className="text-2xl font-bold px-5 text-center">
            Conhe√ßa mais sobre a MEDICALL FARMA
          </p>
          <Link
            className="text-center border w-auto px-8 py-2 rounded-lg hover:font-semibold hover:border-[2px] transition-all"
            href="about"
          >
            Saiba Mais!
          </Link>
        </div>
      </div>
    </AnimationScroll>
  );
};
