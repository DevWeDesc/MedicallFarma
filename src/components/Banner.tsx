"use client";
import React from "react";
// import ImageBanner from "../../public/ImageBanner.png";
import ImageBanner from "../../public/GroupProductsBanner.svg";
import Image from "next/image";
import { motion } from "framer-motion";

export const Banner = () => {
  return (
    <div
      // className="bg-grayDark px-5 flex flex-col-reverse gap-8 py-16 xl:grid xl:grid-cols-2 xl:items-center xl:gap-28 xl:pl-28 xl:pt-24 2xl:pl-80"
      className="bg-grayDark px-5 flex flex-col gap-8 py-16 sm:pt-24 xl:items-center xl:gap-12 xl:pt-24"
    >
      <div className="text-white flex flex-col gap-8">
        <h2 className="max-w-[387px] m-auto text-2xl sm:text-3xl xl:max-w-4xl font-light text-center xl:text-5xl">
          Encontre os melhores Produtos Hospitalares com os Melhores Pre√ßos aqui
          na <span className="font-bold">Medicall Farma!</span>
        </h2>
        {/* <Link
          href="#productRecomended"
          className="xl:max-w-md text-center rounded-full py-3 px-10 font-bold text-xl hover:bg-greenDark transition-all bg-greenDefault"
        >
          Procurar Medicamentos
        </Link> */}
      </div>
      <motion.div
        initial={{ scale: 0 }}
        animate={{ scale: 1 }}
        whileHover={{ scale: 1.1 }}
        transition={{ duration: 1.5 }}
      >
        <Image alt="" className="m-auto" src={ImageBanner} />
      </motion.div>
      {/* <Image alt="" src={ImageBanner} /> */}
    </div>
  );
};
