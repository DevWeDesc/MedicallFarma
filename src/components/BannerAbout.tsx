import Image from "next/image";
import React from "react";
import ImageBannerAbout from "../../public/Rectangle.png";
import ImageBannerAboutMobile from "../../public/ImageAboutMobile.png";

export const BannerAbout = () => {
  return (
    <div className="relative flex flex-col justify-center items-center pt-22">
      <Image
        alt=""
        className="hidden lg:block w-full h-auto"
        src={ImageBannerAbout}
      />
      <Image alt="" src={ImageBannerAboutMobile} className="lg:hidden w-full" />
      <div className="absolute text-white flex flex-col items-center w-full gap-8 px-4">
        <h2 className="max-w-[387px] text-center text-2xl font-bold lg:max-w-4xl lg:m-auto xl:text-5xl">
          Missão de Saúde: Fornecendo Excelência em Medicamentos e Materiais
          Médicos
        </h2>
        <p className="text-sm lg:text-base lg:max-w-4xl lg:m-auto text-center font-light">
          Bem-vindo à MEDICALL FARMA, sua parceira dedicada na entrega de
          soluções de saúde excepcionais. Com uma visão centrada no bem-estar,
          estamos comprometidos em fornecer medicamentos e materiais médicos de
          qualidade superior, promovendo a excelência no cuidado médico.
          Descubra mais sobre nossa história, valores e o compromisso que temos
          em impulsionar a saúde e o progresso na comunidade médica.
        </p>
      </div>
    </div>
  );
};
