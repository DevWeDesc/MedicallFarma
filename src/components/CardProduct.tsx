"use client";
import React, { useEffect, useState } from "react";
import { IProductsData } from "../../types/types";
import Image from "next/image";

export const CardProduct = ({
  image,
  name,
  content,
  supplier,
  categories,
}: IProductsData) => {
  const [configCard, setConfigCard] = useState("");
  const [contentProduct, setContentProduct] = useState("");

  useEffect(() => {
    if (location) {
      location.pathname != "/" && setConfigCard("large");
    }
    configCard === "large"
      ? setContentProduct(content)
      : setContentProduct(content?.substring(0, 83));
  }, [configCard]);

  return (
    <div
      className={`flex flex-col text-left ${
        configCard === "large" ? "gap-4 items-center" : "gap-2"
      } gap-2`}
    >
      <Image alt="" src={image} />
      <div className={`${configCard === "large" && "flex flex-col gap-2"}`}>
        <p
          className={`font-medium min-h-[32px] ${
            configCard === "large" ? "text-2xl" : "text-xs "
          }`}
        >
          {name}
        </p>
        <p
          className={`font-light pr-3 ${
            configCard === "large" ? "text-sm opacity-70" : "text-xs"
          }`}
        >
          {contentProduct}
        </p>
        <p
          className={`font-medium  ${
            configCard === "large" ? "text-base" : "text-xs"
          }`}
        >
          {supplier}
        </p>
        {configCard === "large" && (
          <div>
            <button>Solicitar um Or√ßamento</button>
            <p>
              Categorias:{" "}
              {categories?.map((item, index) => (
                <span key={index}>{item}</span>
              ))}
            </p>
          </div>
        )}
      </div>
    </div>
  );
};
