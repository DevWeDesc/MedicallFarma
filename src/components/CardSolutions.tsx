"use client";
import React, { useEffect, useState } from "react";
import { ICardSoluctions } from "../../types/types";
import Image from "next/image";
import image from "next/image";

export const CardSolutions = ({ image, title }: ICardSoluctions) => {
  const [bgColorCard, setBgColorCard] = useState("");
  useEffect(() => {
    if (title === "Medicamentos") {
      setBgColorCard("bg-blueDark");
    } else if (title === "Dietas Enterais") {
      setBgColorCard("bg-yellowDefault");
    } else {
      setBgColorCard("bg-greenDark");
    }
  }, []);

  return (
    <div
      className={`relative px-2 pt-3 pb-12 rounded-lg ${bgColorCard} hover:bg-opacity-90 transition-all`}
    >
      <p className="text-sm font-semibold">{title}</p>
      <button className="text-[10px] bg-transparent border-b pb-[2px] border-white">
        Conhe√ßa agora
      </button>
      <Image alt="" className="absolute right-2 bottom-2" src={image} />
    </div>
  );
};
