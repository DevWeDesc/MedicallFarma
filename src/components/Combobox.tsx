"use client";
import Image from "next/image";
import Link from "next/link";
import React, { useState } from "react";
import { CategoriesData } from "../../data/data";
import { ICategories } from "../../types/types";
import { CiPlay1 } from "react-icons/ci";

export const Combobox = () => {
  const [comboOpen, setComboOpen] = useState(false);
  const [categorySelected, setCategorySelected] = useState<ICategories>();

  const handleComboOpen = (item: ICategories) => {
    if (comboOpen) {
      setComboOpen(false);
      setCategorySelected(item);
    } else {
      setCategorySelected(item);
      setComboOpen(true);
    }
  };

  return (
    <div className="hidden xl:flex flex-col pt-20 ">
      <div className="bg-grayDark text-white flex gap-10 justify-center">
        {CategoriesData.map((item, index) => (
          <button
            key={index}
            onMouseLeave={() => handleComboOpen(item)}
            onMouseOver={() => handleComboOpen(item)}
            className={`p-8 bg-grayDark hover:font-bold transition-all ${
              categorySelected?.name === item.name && "font-bold"
            }`}
          >
            {item.name}
          </button>
        ))}
      </div>
      <div
        className="relative flex w-full justify-center"
        onMouseLeave={() => {
          setComboOpen(false);
          setCategorySelected(categorySelected);
        }}
        onMouseOver={() => {
          setComboOpen(true);
          setCategorySelected(categorySelected);
        }}
      >
        <div
          className={`${
            comboOpen && categorySelected
              ? "absolute bg-grayLight py-10 px-6 rounded-md grid grid-cols-2 text-white transition-all"
              : "absolute h-0 transition-all overflow-hidden"
          }`}
        >
          <div className="flex flex-col gap-6 ">
            {categorySelected?.subCategories.map((item, index) => (
              <Link
                href={`${categorySelected.name}/${item.title}`}
                key={index}
                className="flex items-center gap-3 opacity-90 hover:opacity-100 hover:font-semibold transition-all"
              >
                <CiPlay1 />
                <p>{item.title}</p>
              </Link>
            ))}
          </div>
          {categorySelected && (
            <Image
              alt=""
              className="max-w-sm"
              src={categorySelected?.image}
            ></Image>
          )}
        </div>
      </div>
    </div>
  );
};
