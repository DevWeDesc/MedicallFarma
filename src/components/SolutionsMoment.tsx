import React from "react";
import { CardSoluctionData } from "../../data/data";
import { CardSolutions } from "./CardSolutions";

export default function SolutionsMoment() {
  return (
    <div className="flex flex-col py-14 items-center justify-center bg-grayDark gap-7 text-white">
      <h2 className="text-2xl font-bold">Soluções do Momento!</h2>
      <div className="w-full grid grid-cols-3 gap-1">
        {CardSoluctionData.map((item) => (
          <CardSolutions
            key={item.title}
            image={item.image}
            title={item.title}
          />
        ))}
      </div>
    </div>
  );
}
