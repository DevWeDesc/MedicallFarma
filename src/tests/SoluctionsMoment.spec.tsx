/**
 * @jest-environment jsdom
 */
import { render } from "@testing-library/react";
import "@testing-library/jest-dom";
import { SolutionsMoment } from "@/components/SolutionsMoment";
import "intersection-observer";

describe("SolutionsMoment render", () => {
  it("renders SolutionsMoment unchanged", () => {
    const { container } = render(<SolutionsMoment />);
    expect(container).toMatchSnapshot();
  });

  it("renders SolutionsMoment link unchanged", () => {
    const { container } = render(<SolutionsMoment />);
    expect(container).toMatchSnapshot();
  });

  it("Solutions Moment Text contain", () => {
    const { getByText } = render(<SolutionsMoment />);

    expect(getByText("Soluções do Momento!")).toBeInTheDocument();
    expect(getByText("Saneantes")).toBeInTheDocument();
    expect(getByText("Tratamentos de ferida")).toBeInTheDocument();
    expect(getByText("Material")).toBeInTheDocument();
  });
});
