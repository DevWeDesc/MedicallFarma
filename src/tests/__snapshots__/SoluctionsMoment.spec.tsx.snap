// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SolutionsMoment render renders SolutionsMoment link unchanged 1`] = `
<div>
  <div
    class="w-full"
    style="opacity: 0; transform: translateY(200px) translateZ(0);"
  >
    <div
      class="flex flex-col py-14 items-center justify-center bg-grayDark gap-7 text-white lg:py-28 lg:px-28 lg:gap-12 2xl:px-0"
    >
      <h2
        class="text-2xl sm:text-3xl font-bold lg:text-5xl"
      >
        Soluções do Momento!
      </h2>
      <div
        class="w-full grid grid-cols-3 gap-1 sm:px-10 sm:gap-3 2xl:px-0 2xl:max-w-7xl 2xl:m-auto"
      >
        <a
          href="/Saneantes"
        >
          <div
            class="h-full flex flex-col justify-between px-2 py-3 rounded-lg bg-blueDark hover:bg-opacity-90 transition-all sm:gap-4 lg:px-9 lg:py-9"
          >
            <p
              class="text-sm sm:text-xl font-semibold lg:h-20 xl:h-auto lg:text-4xl"
            >
              Saneantes
            </p>
            <div
              class="flex items-start justify-between lg:mt-5"
            >
              <button
                class="text-[10px] bg-transparent border-b pb-[2px] border-white sm:text-sm lg:text-lg"
              >
                Conheça agora
              </button>
              <svg
                class="text-4xl lg:text-8xl xl:text-9xl"
                fill="currentColor"
                height="1em"
                stroke="currentColor"
                stroke-width="0"
                viewBox="0 0 24 24"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M19 2V4H17V7C18.6569 7 20 8.34315 20 10V21C20 21.5523 19.5523 22 19 22H5C4.44772 22 4 21.5523 4 21V10C4 8.34315 5.34315 7 7 7V4H5V2H19ZM17 9H7C6.44772 9 6 9.44772 6 10V20H18V10C18 9.44772 17.5523 9 17 9ZM13 11V13H15V15H12.999L13 17H11L10.999 15H9V13H11V11H13ZM15 4H9V7H15V4Z"
                />
              </svg>
            </div>
          </div>
        </a>
        <a
          href="/Tratamentos de ferida"
        >
          <div
            class="h-full flex flex-col justify-between px-2 py-3 rounded-lg bg-yellowDefault hover:bg-opacity-90 transition-all sm:gap-4 lg:px-9 lg:py-9"
          >
            <p
              class="text-sm sm:text-xl font-semibold lg:h-20 xl:h-auto lg:text-4xl"
            >
              Tratamentos de ferida
            </p>
            <div
              class="flex items-start justify-between lg:mt-5"
            >
              <button
                class="text-[10px] bg-transparent border-b pb-[2px] border-white sm:text-sm lg:text-lg"
              >
                Conheça agora
              </button>
              <svg
                class="text-4xl lg:text-8xl xl:text-9xl"
                fill="currentColor"
                height="1em"
                stroke="currentColor"
                stroke-width="0"
                viewBox="0 0 24 24"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M16.5 3C19.5376 3 22 5.5 22 9C22 16 14.5 20 12 21.5C10.0226 20.3135 4.91699 17.563 2.86894 13.001L1 13V11L2.21045 11.0009C2.07425 10.3633 2 9.69651 2 9C2 5.5 4.5 3 7.5 3C9.35997 3 11 4 12 5C13 4 14.64 3 16.5 3ZM16.5 5C15.4241 5 14.2593 5.56911 13.4142 6.41421L12 7.82843L10.5858 6.41421C9.74068 5.56911 8.5759 5 7.5 5C5.55906 5 4 6.6565 4 9C4 9.68542 4.09035 10.3516 4.26658 11.0004L6.43381 11L8.5 7.55635L11.5 12.5563L12.4338 11H17V13H13.5662L11.5 16.4437L8.5 11.4437L7.56619 13L5.10789 13.0006C5.89727 14.3737 7.09304 15.6681 8.64514 16.9029C9.39001 17.4955 10.1845 18.0485 11.0661 18.6038C11.3646 18.7919 11.6611 18.9729 12 19.1752C12.3389 18.9729 12.6354 18.7919 12.9339 18.6038C13.8155 18.0485 14.61 17.4955 15.3549 16.9029C18.3337 14.533 20 11.9435 20 9C20 6.64076 18.463 5 16.5 5Z"
                />
              </svg>
            </div>
          </div>
        </a>
        <a
          href="/Material"
        >
          <div
            class="h-full flex flex-col justify-between px-2 py-3 rounded-lg bg-greenDark hover:bg-opacity-90 transition-all sm:gap-4 lg:px-9 lg:py-9"
          >
            <p
              class="text-sm sm:text-xl font-semibold lg:h-20 xl:h-auto lg:text-4xl"
            >
              Material
            </p>
            <div
              class="flex items-start justify-between lg:mt-5"
            >
              <button
                class="text-[10px] bg-transparent border-b pb-[2px] border-white sm:text-sm lg:text-lg"
              >
                Conheça agora
              </button>
              <svg
                class="text-4xl lg:text-8xl xl:text-9xl"
                fill="currentColor"
                height="1em"
                stroke="currentColor"
                stroke-width="0"
                viewBox="0 0 24 24"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M8 3V5H6V9C6 11.2091 7.79086 13 10 13C12.2091 13 14 11.2091 14 9V5H12V3H15C15.5523 3 16 3.44772 16 4V9C16 11.9727 13.8381 14.4405 11.0008 14.9169L11 16.5C11 18.433 12.567 20 14.5 20C15.9973 20 17.275 19.0598 17.7749 17.7375C16.7283 17.27 16 16.2201 16 15C16 13.3431 17.3431 12 19 12C20.6569 12 22 13.3431 22 15C22 16.3711 21.0802 17.5274 19.824 17.8854C19.2102 20.252 17.0592 22 14.5 22C11.4624 22 9 19.5376 9 16.5L9.00019 14.9171C6.16238 14.4411 4 11.9731 4 9V4C4 3.44772 4.44772 3 5 3H8ZM19 14C18.4477 14 18 14.4477 18 15C18 15.5523 18.4477 16 19 16C19.5523 16 20 15.5523 20 15C20 14.4477 19.5523 14 19 14Z"
                />
              </svg>
            </div>
          </div>
        </a>
      </div>
    </div>
  </div>
</div>
`;

exports[`SolutionsMoment render renders SolutionsMoment unchanged 1`] = `
<div>
  <div
    class="w-full"
    style="opacity: 0; transform: translateY(200px) translateZ(0);"
  >
    <div
      class="flex flex-col py-14 items-center justify-center bg-grayDark gap-7 text-white lg:py-28 lg:px-28 lg:gap-12 2xl:px-0"
    >
      <h2
        class="text-2xl sm:text-3xl font-bold lg:text-5xl"
      >
        Soluções do Momento!
      </h2>
      <div
        class="w-full grid grid-cols-3 gap-1 sm:px-10 sm:gap-3 2xl:px-0 2xl:max-w-7xl 2xl:m-auto"
      >
        <a
          href="/Saneantes"
        >
          <div
            class="h-full flex flex-col justify-between px-2 py-3 rounded-lg bg-blueDark hover:bg-opacity-90 transition-all sm:gap-4 lg:px-9 lg:py-9"
          >
            <p
              class="text-sm sm:text-xl font-semibold lg:h-20 xl:h-auto lg:text-4xl"
            >
              Saneantes
            </p>
            <div
              class="flex items-start justify-between lg:mt-5"
            >
              <button
                class="text-[10px] bg-transparent border-b pb-[2px] border-white sm:text-sm lg:text-lg"
              >
                Conheça agora
              </button>
              <svg
                class="text-4xl lg:text-8xl xl:text-9xl"
                fill="currentColor"
                height="1em"
                stroke="currentColor"
                stroke-width="0"
                viewBox="0 0 24 24"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M19 2V4H17V7C18.6569 7 20 8.34315 20 10V21C20 21.5523 19.5523 22 19 22H5C4.44772 22 4 21.5523 4 21V10C4 8.34315 5.34315 7 7 7V4H5V2H19ZM17 9H7C6.44772 9 6 9.44772 6 10V20H18V10C18 9.44772 17.5523 9 17 9ZM13 11V13H15V15H12.999L13 17H11L10.999 15H9V13H11V11H13ZM15 4H9V7H15V4Z"
                />
              </svg>
            </div>
          </div>
        </a>
        <a
          href="/Tratamentos de ferida"
        >
          <div
            class="h-full flex flex-col justify-between px-2 py-3 rounded-lg bg-yellowDefault hover:bg-opacity-90 transition-all sm:gap-4 lg:px-9 lg:py-9"
          >
            <p
              class="text-sm sm:text-xl font-semibold lg:h-20 xl:h-auto lg:text-4xl"
            >
              Tratamentos de ferida
            </p>
            <div
              class="flex items-start justify-between lg:mt-5"
            >
              <button
                class="text-[10px] bg-transparent border-b pb-[2px] border-white sm:text-sm lg:text-lg"
              >
                Conheça agora
              </button>
              <svg
                class="text-4xl lg:text-8xl xl:text-9xl"
                fill="currentColor"
                height="1em"
                stroke="currentColor"
                stroke-width="0"
                viewBox="0 0 24 24"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M16.5 3C19.5376 3 22 5.5 22 9C22 16 14.5 20 12 21.5C10.0226 20.3135 4.91699 17.563 2.86894 13.001L1 13V11L2.21045 11.0009C2.07425 10.3633 2 9.69651 2 9C2 5.5 4.5 3 7.5 3C9.35997 3 11 4 12 5C13 4 14.64 3 16.5 3ZM16.5 5C15.4241 5 14.2593 5.56911 13.4142 6.41421L12 7.82843L10.5858 6.41421C9.74068 5.56911 8.5759 5 7.5 5C5.55906 5 4 6.6565 4 9C4 9.68542 4.09035 10.3516 4.26658 11.0004L6.43381 11L8.5 7.55635L11.5 12.5563L12.4338 11H17V13H13.5662L11.5 16.4437L8.5 11.4437L7.56619 13L5.10789 13.0006C5.89727 14.3737 7.09304 15.6681 8.64514 16.9029C9.39001 17.4955 10.1845 18.0485 11.0661 18.6038C11.3646 18.7919 11.6611 18.9729 12 19.1752C12.3389 18.9729 12.6354 18.7919 12.9339 18.6038C13.8155 18.0485 14.61 17.4955 15.3549 16.9029C18.3337 14.533 20 11.9435 20 9C20 6.64076 18.463 5 16.5 5Z"
                />
              </svg>
            </div>
          </div>
        </a>
        <a
          href="/Material"
        >
          <div
            class="h-full flex flex-col justify-between px-2 py-3 rounded-lg bg-greenDark hover:bg-opacity-90 transition-all sm:gap-4 lg:px-9 lg:py-9"
          >
            <p
              class="text-sm sm:text-xl font-semibold lg:h-20 xl:h-auto lg:text-4xl"
            >
              Material
            </p>
            <div
              class="flex items-start justify-between lg:mt-5"
            >
              <button
                class="text-[10px] bg-transparent border-b pb-[2px] border-white sm:text-sm lg:text-lg"
              >
                Conheça agora
              </button>
              <svg
                class="text-4xl lg:text-8xl xl:text-9xl"
                fill="currentColor"
                height="1em"
                stroke="currentColor"
                stroke-width="0"
                viewBox="0 0 24 24"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M8 3V5H6V9C6 11.2091 7.79086 13 10 13C12.2091 13 14 11.2091 14 9V5H12V3H15C15.5523 3 16 3.44772 16 4V9C16 11.9727 13.8381 14.4405 11.0008 14.9169L11 16.5C11 18.433 12.567 20 14.5 20C15.9973 20 17.275 19.0598 17.7749 17.7375C16.7283 17.27 16 16.2201 16 15C16 13.3431 17.3431 12 19 12C20.6569 12 22 13.3431 22 15C22 16.3711 21.0802 17.5274 19.824 17.8854C19.2102 20.252 17.0592 22 14.5 22C11.4624 22 9 19.5376 9 16.5L9.00019 14.9171C6.16238 14.4411 4 11.9731 4 9V4C4 3.44772 4.44772 3 5 3H8ZM19 14C18.4477 14 18 14.4477 18 15C18 15.5523 18.4477 16 19 16C19.5523 16 20 15.5523 20 15C20 14.4477 19.5523 14 19 14Z"
                />
              </svg>
            </div>
          </div>
        </a>
      </div>
    </div>
  </div>
</div>
`;
